apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/PathFeatureAssertionsSpec.kt

```diff
@@ -34,5 +34,10 @@ class PathFeatureAssertionsSpec : ch.tutteli.atrium.specs.integration.PathFeatur
 
         a1.parent
         a1 parent {}
+
+        a1 resolve "test"
     }
 }
+
+private fun <T: Path> Expect<T>.resolve(other: String, assertionCreator: Expect<Path>.() -> Unit): Expect<Path> =
```

Comment:
Wasn't able to make the suggestion compile. Hope it's ok this way.

Reply:
I wrote it off the top of my head without checking if it actually compiles. It's good enough for this PR but I am going to rewrite it after the merge. For my taste it resembles too much like a function which is implemented in this API. I'll send you a link to the comment later on

Reply:
https://github.com/robstoll/atrium/commit/8ff3a7495ed5839587da5f35d976184a4a7a4d5e